app/routers/api/paths.py

```
@@ -226,11 +220,7 @@ def get_hot_storage_stats(db: Session = Depends(get_db)):
             logger.exception(f"Error getting disk usage for {path_str}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)